### ISSUE TITLE
Display potential biases on the umap/t-SNE 

### ISSUE THREAD

**Issue by mAGLAVE** [nasqar/seuratv3wizard#11]
Hello,
Thank you for creating this tool. I would like to present it during a course for biologists.
However, I have a problem in the analysis and I want to offer you an idea for improvement.

I do not understand how to know if there is bias to correct. In general, we do a first run without regression of bias, then we display the potential biases on the umap/t-SNE to see if the cells were separated according to this potential bias (in this case it will be necessary to regress this bias) or not (not need to regress it).
For example, here we cannot see the influence of mitochondrial genes on dimension reduction.

In short, I would like to see the groups of genes created during the "QC & Filter", on the umap/t-SNE.
(Or maybe it can be added in the given object to the UCSC CellBrowser, to be display on it.)

In addition, another possible improvement would be the addition of the cell cycle assessment by seurat: https://satijalab.org/seurat/v3.1/cell_cycle_vignette.html

Thank you!
